### PR TITLE
Added the policy name to cost-savings

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description
Replace policy name in cost-savings tag.

### CODE REVIEW

**Review by ebattat — CHANGES_REQUESTED** [review]
/LGTM

File: cloud_governance/policy/helpers/aws/aws_policy_operations.py [hunk 1]
```diff
@@ -27,7 +28,7 @@ def __init__(self):
         self._ec2_operations = EC2Operations(region=self._region)
         self._cloudwatch = CloudWatchOperations(region=self._region)
         self._resource_pricing = ResourcesPricing()
-        self.cost_savings_tag = [{'Key': 'cost-savings', 'Value': 'true'}]
+        self.cost_savings_tag = [{'Key': 'cost-savings', 'Value': self.policy_name}]
```
Comment (ebattat) [inline cloud_governance/policy/helpers/aws/aws_policy_operations.py:31]: Do you think it better to create another tag ?
policy:policy_name

Comment (athiruma) [reply cloud_governance/policy/helpers/aws/aws_policy_operations.py:31]: No need.
 cost-savings: policy_name specifies the savings amount on that policy.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**